src/routes/api/voting.js

```diff
@@ -15,87 +15,92 @@ module.exports = function(express) {
  * This sets the util voting to equal the path to the lib/util
 */
 
-  /** Read One. */
-  router.get('/voting/:vote_id', function(req, res) {
 /**
  * @function get
- * @description Gathers Data
- * @param req 
- * Requests Data
+ * @description This gathers data.
+ * @param req
+ * This requests Data.
  * @param res
- * Listens for response  
+ * This listens for response.
 */
+  /** Read One. */
+  router.get('/voting/:vote_id', function(req, res) {
+
+    /**
+     * @property req.body
+     * @description Holds parameters that are sent up from the client as part of a POST request.
+    */
+    /**
+     * @property {attribute} id
+     * Defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking.
+    */
+    /**
+     * @property req.params
+     * @description This will return parameters in the matched route.
+    */
     req.body.id = req.params.vote_id;
-/**
- * @property req.body
- * @description holds parameters that are sent up from the client as part of a POST request 
-*/
-/**
- * @property {attribute} id
- * Defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking
-*/
-/**
- * @property req.params
- * @description will return parameters in the matched route
-*/
 
-    voting.find(req.body, function(err) { 
-/**
- * @function find
- * @description Finds ID in DB using _find in models/voting.js
- * @param req.body
- * Requests the body information
-*/
+    /**
+     * @function find
+     * @description Finds ID in DB using _find in models/voting.js.
+     * @param req.body
+     * This requests the body information.
+    */
+    voting.find(req.body, function(err) {
+
       /** ERROR Encountered. */
       res.status(500).json(err);
     }, function(data) {
       res.status(200).json(data);
     });
   });
 
-
+  /**
+   * @function get
+   * @description This gathers data.
+   * @param req
+   * This requests data.
+   * @param res
+   * This listens for response.
+  */
```

Comment:
As we discussed the routes should not have the full function description treatment with respect to JS docs. Please continue to remove all of these.